app/rubin-2025/register-form.jsx

```diff
@@ -79,233 +79,294 @@ class RegisterForm extends React.Component {
-              <Translate className="form-help info" content="registerForm.emailInvalidFormat" />
-            )}
+        </div>
+
+        <div className="form-row">
+          <div className="label-block">
+            <label htmlFor="register-form-email">
+              {this.state.underAge
+                ? <Translate content="registerForm.guardianEmail" />
+                : <Translate content="registerForm.email" />
+              }
+            </label>
+
+            <div aria-live="polite" id="register-form-email-help-message-1">
+              {emailInvalidChars && (
+                <Translate className="form-help error" content="registerForm.emailInvalidChars" />
+              )}
+            </div>
+
+            <div aria-live="polite" id="register-form-email-help-message-2">
+              {(!emailInvalidChars && emailInvalidFormat) && (
+                <Translate className="form-help info" content="registerForm.emailInvalidFormat" />
+              )}
+            </div>
+
             {'emailConflict' in this.state.pending && (
               <LoadingIndicator />
             )}
-            {emailConflict && (
-              <span className="form-help error">
-                <Translate content="registerForm.emailConflict" />{' '}
-                <a href={`${window.location.origin}/reset-password`}>
-                  <Translate content="registerForm.forgotPassword" />
-                </a>
-              </span>
-            )}
+
+            <div aria-live="polite" id="register-form-email-help-message-3">
+              {emailConflict && (
+                <span className="form-help error">
+                  <Translate content="registerForm.emailConflict" />{' '}
+                  <a href={`${window.location.origin}/reset-password`}>
+                    <Translate content="registerForm.forgotPassword" />
+                  </a>
+                </span>
+              )}
+            </div>
+
             {(this.state.input_email?.length > 0
               && !emailConflict
               && !emailInvalidChars
               && !emailInvalidFormat
+              && !('emailConflict' in this.state.pending)
             ) && (
               <Translate className="form-help success" content="registerForm.looksGood" />
             )}
+
             <Translate className="form-help info right-align" content="registerForm.required" />
-          </span>
+          </div>
+
           <input
-            type="text"
-            name="email"
-            id="register-form-email"
-            value={this.state.input_email}
+            aria-describedby="register-form-error-message register-form-email-help-message-1 register-form-email-help-message-2 register-form-email-help-message-3"
             className="standard-input full"
             disabled={inputDisabled}
+            id="register-form-email"
+            name="email"
             onChange={this.handleUserInput}
+            type="text"
```

Comment:
```suggestion
            type="email"
            autocomplete="email"
```

NB. `type="email"` gives you email address pattern-matching for free.